### PR TITLE
Learn page

### PR DESCRIPTION
**Pull request by GeoStefan**
This is the learn page prototype. It has a description section and three columns for courses.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
 <head>
 	<link rel="stylesheet" type="text/css" href="style.css">
 	<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="author" content="Popescu Georgian Stefan">
```
Comment (bogsen) [inline index.html]: We're working on this as a team 🙂

File: style.css [hunk 2]
```diff
@@ -52,9 +56,164 @@ body > .container {
...
+
 .box {
 	background-color: white;
 	box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.25);
 	border-radius: 6px;
+	height: 10rem;
+}
+
+#description-image {
+	float: rigth;
```
Comment (bogsen) [inline style.css]: Typo

File: style.css [hunk 3]
```diff
@@ -52,9 +56,164 @@ body > .container {
...
+}
+
+.description2 {
+	padding: 1rem;
+	float: left;
+	height: 8rem;
+	max-width: 70%;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	position:relative;
```
Comment (bogsen) [inline style.css]: Missing space

File: style.css [hunk 4]
```diff
@@ -52,9 +56,164 @@ body > .container {
...
+		max-width: 70rem;
+	}
+
+	.description2 {
+		height: 8rem;
+		width: 55rem;
+	}
+
+	#description-image {
+	}
```
Comment (bogsen) [inline style.css]: Do we need this empty selector?

**Review by bogsen — CHANGES_REQUESTED** [review]
It's good! 😀 but please make the indentation consistent in both the HTML and CSS files

File: style.css (hunk 4 above)
Comment (GeoStefan) [reply style.css]: Not now.

**Review by bogsen — APPROVED** [review]

**Pull request merged**